src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/ReferencePopulationDefinitionFreeText.java

```diff
@@ -0,0 +1,84 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import org.apache.commons.lang3.StringUtils;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+import java.util.Arrays;
+
+/**
+ * Example: {@code FTX+RPD+++Equivocal'}
+ */
+@EqualsAndHashCode(callSuper = true)
+@Getter
+@Setter
+@AllArgsConstructor
+public class ReferencePopulationDefinitionFreeText extends Segment {
+    private static final String KEY = "FTX";
+    private static final String QUALIFIER = "RPD";
+    private static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+
+    private static final int INDEX_FREE_TEXTS = 4;
+    private static final int MAXIMUM_FREE_TEXTS = 5;
+
+    @NonNull
+    private String[] freeTexts;
+
+    public static ReferencePopulationDefinitionFreeText fromText(final String edifact) {
```

Comment:
Why are we using a different method name for converting an EDIFACT string to a Java object? 

I'm slightly wary of doing this in case we rely on the name `fromString` but if everyone else likes this name, I'm happy with it - more a point of discussion than an actual issue

Reply:
That's just a mistake - the segment is xxxFreeText so "text" must have been in my mind when writing it!